CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 Notable changes to the framework should be documented here
 
 3.12.3:
+    - Extended watchdog for AIX (ENT-4995)
```

Comment:
all other entries are prefixed with TAB, and this one - with 4 spaces.
I believe TAB is standard in this file.

Reply:
fixed!